.circleci/config.yml

```diff
@@ -22,6 +23,10 @@ jobs:
       - run: |
           $NYC yarn test
           curl -s https://codecov.io/bash | bash
+  node-10:
+    <<: *test
+    docker:
+      - image: chadian/nsis:10-2
```

Comment:
An `oclif/nsis` image needs to be pushed with the changes reflected by this `chadian/nsis:10-2`, built from this [Dockerfile](https://github.com/oclif/docker/pull/1/files#diff-baf77feed21610604ea0d03b1ec5b665).
Here's an example of a successful [CI test run](https://circleci.com/gh/oclif/dev-cli/1167) using this image.